File: 1006.c
#include<stdio.h>
 
int main(){
 
    char first[10], second[10], thirth[10];
    scanf("%s", &*first);
    scanf("%s", &*second);
    scanf("%s", &*thirth);
 
    if(first[0]=='b' && second[0]=='b' && thirth[0]=='g')
        printf("black\nblack\nGREEN\n");
    else if(first[0]=='b' && second[0]=='b' && thirth[0]=='G')
        printf("black\nyellow\nblack\n");
    else if(first[0]=='b' && second[0]=='y' && thirth[0]=='b')
        printf("red\nblack\nblack\n");
    else if(first[0]=='r' && second[0]=='b' && thirth[0]=='b')
        printf("red\nyellow\nblack\n");
    else if(first[0]=='r' && second[0]=='y' && thirth[0]=='b')
        printf("black\nblack\ngreen\n");
    else if(first[0]=='b' && second[0]=='Y' && thirth[0]=='b')
        printf("black\nYELLOW\nblack\n");
    else if(first[0]=='R' && second[0]=='b' && thirth[0]=='b')
        printf("RED\nblack\nblack\n");
    else if(first[0]=='b' && second[0]=='b' && thirth[0]=='G')
        printf("black\nblack\nGREEN\n");
    else printf("error");  
 
    return 0;
}

File: 1006.py
a = input()
b = input()
c = input()
 
if a[0] == 'b' and b[0] == 'b' and c[0] == 'g':
    print("black\nblack\nGREEN", end = "")
elif a[0] == 'b' and b[0] == 'b' and c[0] == 'G':
    print("black\nyellow\nblack", end = "")
elif a[0] == 'b' and b[0] == 'y' and c[0] == 'b':
    print("red\nblack\nblack", end = "")
elif a[0] == 'r' and b[0] == 'b' and c[0] == 'b':
    print("red\nyellow\nblack", end = "")
elif a[0] == 'r' and b[0] == 'y' and c[0] == 'b':
    print("black\nblack\ngreen",end = "")
elif a[0] == 'b' and b[0] == 'Y' and c[0] == 'b':
    print("black\nYELLOW\nblack", end = "")
elif a[0] == 'R' and b[0] == 'b' and c[0] == 'b':
    print("RED\nblack\nblack", end = "")
elif a[0] == 'b' and b[0] == 'b' and c[0] == 'G':
    print("black\nblack\nGREEN", end = "")
elif a[0] == 'r' and b[0] == 'y' and c[0] == 'g':
    print("error")
else:
    print("error\n")
